bundles/CoreBundle/Resources/translations/de.json

```diff
@@ -52,6 +52,8 @@
     "send_newsletter": "Newsletter JETZT versenden",
     "object_filter": "Objekt Filter",
     "add_newsletter": "Newsletter hinzufügen",
+    "plain_text": "Klartext",
```

Comment:
Languages other than EN are managed here: 
Please revert this change and add the translation in POEditor once this PR is merged.

Reply:
Where exactly? Link missing...

Reply:
https://poeditor.com/join/project/VWmZyvFVMH

![image](https://user-images.githubusercontent.com/142037/62353662-6ed17800-b50b-11e9-8f05-6f0ab55a11c0.png)
![image](https://user-images.githubusercontent.com/142037/62353687-801a8480-b50b-11e9-966c-82545ec47440.png)

Reply:
Thank you, will do